raiden/constants.py

```diff
@@ -175,6 +175,10 @@ class RoutingMode(Enum):
 BLOCK_ID_LATEST: Literal["latest"] = "latest"
 BLOCK_ID_PENDING: Literal["pending"] = "pending"
 
+# Thresholds for the ``eth.getLogs`` call. Used to automatically adjust the block batch size.
+ETH_GET_LOGS_TIMEOUT = 10
```

Comment:
Putting the unit in a comment would remove doubts about the unit of that value.

Reply:
Yes, we should probably do that for all the settings / constants we have.
I'll open an issue for it.